apps/client-web/src/common/utils/paths.ts

```diff
@@ -1,6 +1,6 @@
-import { MashcardContext } from '../mashcardContext'
+import { currentPodUsername } from './currentPodUsername'
 
 export const rootPath = (context: MashcardContext): string =>
   context.lastDomain && context.lastBlockIds && context.lastBlockIds[context.lastDomain!]
     ? `/${context.lastDomain}/${context.lastBlockIds[context.lastDomain!]}`
```

Comment:
FYI lastDomain / lastBlockIds no longer works (not getting saved). I think it's related to the refactor of block sync from @aligo

Reply:
We can remove `lastDomain` and `lastBlockIds` when `SyncBatch` and `BlockInfo` is removed.